tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -110,8 +110,10 @@ def test_fio_workload_simple(self, ripsaw, interface, io_pattern):
         # Todo: have pvc_size set to 'get_osd_pods_memory_sum * 5'
         #  once pr-2037 is merged
         fio_cr['spec']['clustername'] = config.ENV_DATA['platform'] + get_build() + get_ocs_version()
-        fio_cr['spec']['test_user'] = interface + io_pattern
+        fio_cr['spec']['test_user'] = get_ocs_version() + interface + io_pattern
```

Comment:
I think it will be better as :
`        fio_cr['spec']['test_user'] = '_'.joine([get_ocs_version(), interface, io_pattern])`